### DISCUSSION TITLE
AXP2101

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#22501] [Q&A]
i have th LilyGo T-Watch S3.
It has a AXP2101 for powermanagment. Is it compatible with Tasmota?
the only thing i found was a depricated [thing](https://github.com/arendst/Tasmota/tree/682129381983a3f1e1c9cb0de20a017a0704bf06/lib/lib_deprecated/TTGO_TWatch_Library)
 need to check the voltage of the battery at least.
thanks!

**Comment by Jason2866** [comment]
See https://github.com/arendst/Tasmota/tree/development/tasmota/berry/drivers for currently available drivers for AXPxxx chips
